packages/components/src/components/hds/form/field/index.ts

```diff
@@ -131,10 +132,10 @@ export default class HdsFormField extends Component<HdsFormFieldSignature> {
 
   @action
   appendDescriptor(element: HTMLElement): void {
-    registerAriaDescriptionElement(this, element);
+    registerAriaDescriptionElement(this as AriaDescribedByComponent, element);
```

Comment:
`AriaDescribedByComponent` is a more generic component signature covering the Field and Fieldset components (both potentially having descriptors, such as HelperText and Error). Their signatures vary in a sense that `@layout` has different values, so we're doing this casting here to ensure TypeScript that they are matching the more generic type. I couldn't find an alternative, but open to suggestions.